js/modules/k6/http/http_request.go

```diff
@@ -553,7 +545,8 @@ func (h *HTTP) request(ctx context.Context, preq *parsedHTTPRequest) (*HTTPRespo
 		}
 		_ = res.Body.Close()
 	}
-	trail := tracer.Done()
+	trail := transport.(*netext.Transport).GetTrail()
```

Comment:
Hmm won't this fail when we use NTLM? Since `transport` then won't actually have the `*netext.Transport` type?

Reply:
Yeah, this causes panic even with super-simple `http.get("http://test.loadimpact.com", { auth: "ntlm" });` requests...